cmd/plugin.go

```diff
@@ -411,17 +415,18 @@ func runPluginWorker(ctx context.Context, upload bool, workChan <-chan PluginTra
 	for {
 		select {
 		case <-ctx.Done():
-			return ctx.Err()
+			return lastAttemptedSource, ctx.Err()
 		case transfer, ok := <-workChan:
 			if !ok {
 				tc.Close()
 				workChan = nil
 				break
 			}
+			lastAttemptedSource = transfer.url.String()
```

Comment:
What's returned here?  Is it the URL that was used for the transfer or the URL that was requested by HTCondor?